### ISSUE TITLE
Bug report: Site is for QA purposes but is indexable on Google

### ISSUE THREAD

**Issue by adriancooke** [CivicActions/accessibility#852]
**Describe the bug**

While as a user I expected to see only the canonical version of the CA Accessibility site in search results, I saw results for the QA site which makes me doubt its reliability and relevance. 

Personas: Engineer Emery, QA Quincy

**To Reproduce**
1. Go to www.google.com
2. Type civicactions accessibility developer
3. Scroll down to the Developer result
4. Observe the hostname is accessibility-qa.civicactions.com
5. Authenticate to Google and open Google Search Console
6. Navigate to Indexing > Pages > Duplicate without user-selected canonical
7. Observe that most URLs in the report have the accessibility-qa hostname

**Expected behavior**
CA Accessibility results in Google should include the accessibility.civicactions.com hostname and should not include accessibility-qa.civicactions.com.

In Google Search Console’s Duplicate without user-selected canonical report, there should be no accessibility-qa hostname results because the crawler should see that they are not indexable.

**Screenshots**

<img width="2680" height="1892" alt="google query for civicactions accessibility designer on Nov 17, 2025 which includes the qa site as the first organic result" src="https://github.com/user-attachments/assets/19264dfd-21f2-4cd2-95e4-98349469a4f7" />
<img width="2686" height="2134" alt="google search console Duplicate without user selected canonical report on Nov 17, 2025 showing many accessibility-qa.civicactions.com results" src="https://github.com/user-attachments/assets/ef388365-6ac8-4dfa-b35b-4aac5da70ba3" />

**Desktop (please complete the following information):**
 - OS: macOS 15.7.2
 - Browser: Firefox 
 - Version: 145.0

**Additional context**

My suggestion is to allow crawl in robots.txt and don’t disallow it in metadata, but disallow _indexing_ using:
`<meta name="robots" content="noindex">`. For more information see 

* [Robots.txt versus noindex (Confluence)](https://civicactions.atlassian.net/wiki/spaces/DES/pages/583335951/Robots.txt+versus+noindex) and 
* [Robots meta tag, data-nosnippet, and X-Robots-Tag specifications (Google Search Central)](https://developers.google.com/search/docs/crawling-indexing/block-indexing)

**Comment by dmundra** [comment]
Hey @adriancooke, I submitted a PR #854 to fix it. I also added the QA site to google search console so I can request removal of all results. Hopefully those two things can fix this. Can you take a look when you get a chance please?
